nbconvert/templates/latex/base.tplx

```diff
@@ -13,7 +13,29 @@ This template does not define a docclass, the inheriting class must define this.
     ((* block docclass *))((* endblock docclass *))
     
     ((* block packages *))
-    \usepackage{graphicx} % Used to insert images
+    % Nicer default font than Computer Modern for most use cases
+    \usepackage{palatino}
+
+    % Basic figure setup, for now with no caption control since it's done
+    % automatically by Pandoc (which extracts ![](path) syntax from Markdown).
+    \usepackage{graphicx}
+    % We will generate all images so they have a width \maxwidth. This means
+    % that they will get their normal width if they fit onto the page, but
+    % are scaled down if they would overflow the margins.
```

Comment:
Mmh, it wasn't working for me... The reason I introduced this logic was precisely because when I started creating the Sloan report, I was getting all my figures to be absolutely huge, blowing way past the page borders.  I'm on OS X, pandoc 1.15.2.1.

So, evidently, something wasn't quite right until I put this logic in, but we should sort it out first...